test/func/tests/package.json

```diff
@@ -3,18 +3,18 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "gui:testplane-common": "PROJECT_UNDER_TEST=testplane npx testplane --set common gui",
-    "gui:testplane-eye": "PROJECT_UNDER_TEST=testplane-eye npx testplane --no --set eye gui",
-    "gui:testplane-gui": "PROJECT_UNDER_TEST=testplane-gui npx testplane --no --set common-gui gui",
+    "gui:testplane-common": "TOOL=testplane PROJECT_UNDER_TEST=testplane npx testplane --set common gui",
+    "gui:testplane-eye": "TOOL=testplane PROJECT_UNDER_TEST=testplane-eye npx testplane --no --set eye gui",
+    "gui:testplane-gui": "TOOL=testplane PROJECT_UNDER_TEST=testplane-gui npx testplane --no --set common-gui gui",
     "gui:playwright": "PROJECT_UNDER_TEST=playwright npx testplane --set common gui",
-    "gui:plugins": "PROJECT_UNDER_TEST=plugins SERVER_PORT=8084 npx testplane --set plugins gui",
-    "gui:testplane-tinder": "PROJECT_UNDER_TEST=testplane-gui SERVER_PORT=8084 npx testplane --set common-tinder gui",
-    "testplane:testplane-common": "PROJECT_UNDER_TEST=testplane SERVER_PORT=8061 npx testplane --set common",
-    "testplane:testplane-eye": "PROJECT_UNDER_TEST=testplane-eye SERVER_PORT=8062 npx testplane --set eye",
-    "testplane:testplane-gui": "PROJECT_UNDER_TEST=testplane-gui SERVER_PORT=8063 npx testplane --no --set common-gui",
+    "gui:plugins": "TOOL=testplane PROJECT_UNDER_TEST=plugins SERVER_PORT=8084 npx testplane --set plugins gui",
+    "gui:testplane-tinder": "TOOL=testplane PROJECT_UNDER_TEST=testplane-gui SERVER_PORT=8084 npx testplane --set common-tinder gui",
+    "testplane:testplane-common": "TOOL=testplane PROJECT_UNDER_TEST=testplane SERVER_PORT=8061 npx testplane --set common",
+    "testplane:testplane-eye": "TOOL=testplane PROJECT_UNDER_TEST=testplane-eye SERVER_PORT=8062 npx testplane --set eye",
+    "testplane:testplane-gui": "TOOL=testplane PROJECT_UNDER_TEST=testplane-gui SERVER_PORT=8063 npx testplane --no --set common-gui",
     "testplane:playwright": "PROJECT_UNDER_TEST=playwright SERVER_PORT=8065 npx testplane --set common",
```

Comment:
Maybe we should provide `TOOL=playwright` to pwt tests too?
If i saw `if (process.env.TOOL == "testplane")`, i wouldn't expect it to be `undefined` for pwt tests